Makefile

```diff
@@ -16,10 +16,14 @@ export CONTAINER_REGISTRY := ""
 # Openshift
 export APP_NAME:=tcloud
 export OS_NAMESPACE_PREFIX:=cd4869
-export OS_NAMESPACE_SUFFIX?=dev
+export OS_NAMESPACE_SUFFIX?=test
```

Comment:
Should we set this back to dev just to be safe (ie in case we're actually intending to deploy changes to dev and forget to change it back?)